### DISCUSSION TITLE
SequentialChains

### DISCUSSION THREAD

**Discussion by imatenaer** [langflow-ai/langflow#468] [Q&A]
I know that in LangChain Sequential Chains are supported. Is it possible to create Sequential Chains in LangFlow?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
